src/Agent.Plugins/TfsVCCliManager.cs

```diff
@@ -257,6 +257,29 @@ protected enum FormatFlags
             OmitNoPrompt = 4,
             All = OmitCollectionUrl | OmitLogin | OmitNoPrompt,
         }
+
+        /*
```

Comment:
can we add a comment as to why we are leaving a bunch of methods commented out?

Reply:
Whoops that was a mistake. Deleting.